ast/misc.go

```diff
@@ -101,7 +101,15 @@ type TraceStmt struct {
 
 // Restore implements Node interface.
 func (n *TraceStmt) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	ctx.WriteKeyWord("TRACE ")
```

Comment:
we should add test here:
https://github.com/pingcap/parser/blob/7b0cfe281af20fb68fe83bd5d66c03c6dcfb6d9d/parser_test.go#L2297-L2304

but we need the implement of `dmlstmt`s to test `TraceStmt`

Reply:
all of `dmlstmt`s were implemented, you can test this pr now.